src/Form/test-cases/steam_checkout.html

```diff
@@ -0,0 +1,1993 @@
+                                                name="last_name" id="last_name" value="" autocomplete="cc-family-name"
+                                                data-manual-scoring="lastName">
+                                        </div>
+
+                                        <div class="form_row">
+                                            <label for="billing_address">Billing address</label>
+                                            <input tabindex="203" type="text" class="text" maxlength="128"
+                                                name="billing_address" id="billing_address" value=""
+                                                autocomplete="address-line1" data-manual-scoring="addressStreet">
+                                        </div>
+
+                                        <div class="form_row">
+                                            <label for="billing_address_two">Billing address, line 2</label>
+                                            <input tabindex="204" type="text" class="text" maxlength="128"
+                                                name="billing_address_two" id="billing_address_two" value=""
+                                                autocomplete="address-line2" data-manual-scoring="addressStreet2">
+                                        </div>
+
+
+                                    </div>
+                                    <div class="form_right">
+                                        <div class="form_row">
+                                            <label for="billing_city">City</label>
+                                            <input type="text" class="text" tabindex="205" maxlength="50"
+                                                name="billing_city" id="billing_city" value=""
+                                                autocomplete="address-level2" data-manual-scoring="addressCity">
+                                        </div>
+
+                                        <div id="billing_state_input" class="form_row">
+                                            <label id="billing_state_label" for="billing_state">State/Province</label>
+                                            <input tabindex="207" type="text" class="text" maxlength="33"
+                                                name="billing_state_text" id="billing_state_text" value=""
+                                                style="display: none;" data-manual-scoring="unknown">
+                                            <div class="dselect_container" id="billing_state_select_dselect_container"
+                                                style="">
+                                                <input id="billing_state_select" type="hidden"
+                                                    name="billing_state_select" value="" data-manual-scoring="addressProvince">
```

Comment:
Here and in other places in the form, Steam re-implements a `<select>` field in HTML, and then has two hidden inputs associated with it. It's not clear to me what (if anything) should be autofilled here.